src/main/kotlin/no/nav/familie/ef/sak/oppgave/OppgaveService.kt

```diff
@@ -155,7 +158,11 @@ class OppgaveService(
     }
 
     private fun ferdigstillOppgaveOgSettEfOppgaveTilFerdig(oppgave: EfOppgave) {
-        ferdigstillOppgave(oppgave.gsakOppgaveId)
+        val gsakOppgave = oppgaveClient.finnOppgaveMedId(oppgaveId = oppgave.gsakOppgaveId)
+        val gsakOppgaveStatus = gsakOppgave.status ?: error("Kunne ikke finne status for gsak oppgave")
+        if (!gsakOppgaveStatus.equals(StatusEnum.FEILREGISTRERT)) {
```

Comment:
Har dere sjekket med fag at dette er ønskelig måte å håndtere det på? 
Som beskrevet på slack så "mister" man den litt i oppgaveoversikten om hva som er utført på personen hvis den er feilregistrert.

Reply:
Fint hvis det logges om den allerede er feilregistrert

Reply:
Ikke avklart med fag - dette har skjedd én gang i EF (flere i BAKS). 
Og de må ha en mulighet til å ferdigstillle hos oss, tenker jeg. 

Og på sikt bør jo vi "overta" visning av oppgavehistorikken.  
Så implikasjonene av denne er marginal, tenker jeg.

Reply:
I dette tilfelle er det sikkert greit då de prøver å henlegge behandlingen, men nå åpner vi strengt tatt opp for at det skal være mulig å ferdigstille i gosys, hvilket egentlige ikke ønskes.
Det er mulig det riktige her er at den patches i gosys, og at vi ikke åpner opp 👀 
Samtidig så tror jeg ikke de er veldig glade over å patche 1 hendelse

Reply:
Så du tenker at vi burde avklare dette ?

Reply:
Kan strekke meg til å være enig med @blommish her  - at vi tillater henleggelse iallefall :) Så får vi heller ta opp diskusjonen når vi får en case med feilregistrert oppgave som skal gjennomføres.. :)

Reply:
I stedet for å sjekke oppgaven for hver lukking så endret jeg til try catch. 
Ulempen er at jeg endret til om meldingen inneholder at oppgaven er feilregistrert, men tenker det kanskje er greit i disse tilfeller? Hvis ikke så kan vi eventuellt hente oppgaven/status i catch?